### ISSUE TITLE
upvote pyIDM!

### ISSUE THREAD

**Issue by pyIDM** [pyIDM/PyIDM#30]
hello everyone if you like this project please give it a thumbs up at 
https://github.com/vinta/awesome-python/pull/1450

with enough upvotes this project will be listed on the Vinta Awesome python Applications

**Comment by PySimpleGUI** [comment]
Posted a request over in PySimpleGUI announcements today.  I see a couple more have been added already.  Hopefully you'll get over the total soon.  Don't be shy about asking people that have been getting lots of support for their help ;-)  It's one way people "can contribute or give back" that want to help.

**Comment by pyIDM** [comment]
Well that's a great boost, thanks a lot for this support :)

**Comment by PySimpleGUI** [comment]
Yohoo!  You're at 15 right now.  Only need to get to 20.  Nothing in the rules that say those weird downvotes count against you... only that you need to upvotes.   Some people just like to see stuff burn.  Keep on making these cool ideas you have!

**Comment by pyIDM** [comment]
WONDERFUL :smile:

**Comment by PySimpleGUI** [comment]
## 18.... only TWO to go!

**Comment by pyIDM** [comment]
We’re getting there 👍👍

**Comment by pyIDM** [comment]
@PySimpleGUI 
:tada: **Just reached 20 upvotes on vinta**  :tada:

**Comment by PySimpleGUI** [comment]
There ya go!
SEE.... patience.  Let the internet work its magic.

It took a little while.  It's a medium term investment.  Looks like I opened it on Dec 28th.

Now begins the pestering process to get the change integrated.

I forget who I bothered before to get PySimpleGUI integrated.  It took longer to get added to the list officially than it did to get the votes.

I'll add a comment and see if it gets noticed by anyone.
